### ISSUE TITLE
Component: p-listbox missing aria-label

### ISSUE THREAD

**Issue by WTW-Jacob-Evans** [primefaces/primeng#15084]
### Describe the bug

Applying accessibility standards, the list within the p-listbox component appears to be missing aria-labels / accessible text, which is being highlighted by Axe Dev Tools.
![image](https://github.com/primefaces/primeng/assets/163871134/a61336dd-3cad-4bf5-b2c9-eb143fc09869)
![image](https://github.com/primefaces/primeng/assets/163871134/0454a374-46f9-467e-9338-d2b649390eee)

### Environment

Windows 10,
Chrome,
Angular 17.2.3,
Primeng 17.10.0

### Angular version

17.2.3

### PrimeNG version

17.10.0

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

18.18.0
